shapeshift/web
# refactor: re-word wallet successfully paired screen

## Description

Fixes #856

## Notice

Before submitting a pull request, please make sure you have answered the following:


## Pull Request Type


## Issue (if applicable)

closes #856

## Testing

1. Setup local dev environment
2. Connect a Metamask/Portis wallet 
3. Notice the new wording
